cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec *rspec.Spec) error {
+	fmt.Println("validating readonlyPaths")
+	for _, v := range spec.Linux.ReadonlyPaths {
+		fi, err := os.Stat(v)
```

Comment:
This stat-based approach should be changed to an [attempted-write approach][1].

[1]: https://github.com/opencontainers/ocitools/pull/78#issuecomment-221837970